common-npm-packages/packaging-common/npm/npmregistry.ts

```diff
@@ -25,6 +25,7 @@ export class NpmRegistry implements INpmRegistry {
         this.authOnly = authOnly || false;
     }
 
+    /** Return NpmRegistry with masked auth from Service Endpoint. */
```

Comment:
nit: why multiline comment when its a single line?

Reply:
I think this is the syntax for method docs.